types/emscripten/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
     "compilerOptions": {
+        "target": "esnext",
         "module": "commonjs",
         "lib": [
-            "es6",
-            "es2020",
+            "esnext",
```

Comment:
Please change these to a specific version or leave it as-is instead of esnext since that changes from year to year

Reply:
Done